app/gallery/page.tsx

```diff
@@ -0,0 +1,21 @@
+"use client"
+import type { NextPage } from 'next';
+import '@/src/styles/reset.scss';
+
+import PhotoGallery from '@/src/sections/Photo-Gallery';
+import NavigationBar from '@/src/components/navbar';
+import Footer from '@/src/components/footer'
+
+// here we will compile all the sections of the website together
+
+const GalleryPage: NextPage = () => {
+  return (
+    <main>
+      <NavigationBar />
+      <PhotoGallery />
+      <Footer />
```

Comment:
Could you add margins between the Photo Gallery and Footer? It looks a little crammed
![image](https://github.com/acmucsd/projects-website/assets/96843411/131e22ef-7e6c-44e1-9570-f8df34422e89)